internal/backends/winit/glwindow.rs

```diff
@@ -202,7 +202,9 @@ impl<Renderer: WinitCompatibleRenderer + 'static> WinitWindow for GLWindow<Rende
 
         self.pending_redraw.set(false);
 
-        self.renderer.render(&window.canvas);
+        let inner_size = window.window.inner_size();
+        let physical_size = PhysicalSize::new(inner_size.width, inner_size.height);
```

Comment:
A bit sad that we lose type safety there.  Looking at this snippet of code, i don't know that inner_size is indeed a physical size.  Maybe the type could be made explicit.  Or going through a conversion function that do the conversion in a type safe manner.

But feel free to ignore.

Reply:
No no, you have a valid point! I can clean this up with a helper trait I think, that converts winit’s physical size to our physical. Then it’s safe and less LOC here.